### DISCUSSION TITLE
Suggestions from Pylint.

### DISCUSSION THREAD

**Discussion by deepgandhi** [LewisGaul/minegauler#182] [Q&A]
While running the Pylint on minegauler/shared/utils.py, it shows that encoding is not specified for reading and writing a file. Ref line 316.

My question is whether is there any standard encoding which is required to specify? And If yes, then which encoding is required to use.

I will update the specified encoding at all places suggested by Pylint and will submit the PR.

**Comment by LewisGaul** [comment]
I believe we want "utf-8", see https://docs.python.org/3/howto/unicode.html. A PR for this sounds good, thanks :)

**Comment by deepgandhi** [comment]
I have made sample correction in minegauler/core/controller.py and submit the pull request #183. If it is okay, then i will proceed with all other files.
